### ISSUE TITLE
IBE bugs out and reports "the server doesn't have IBE installed" even though I'm on singleplayer.

### ISSUE THREAD

**Issue by the3dbb** [Franckyi/IBE-Editor#9]
### Specifications
* Operating system: Windows 10 Home (x64, 64-bit), version 1909, with 8GB of RAM.
* Minecraft Version: 1.13.2 (Modded / Forge)
* Launcher: MultiMC 5 / 0.6.11-1430 / x86 (32-bit)
* IBE Version: 1.0.1 for 1.13.2

### Problem
> With the specifications above, when I boot into Minecraft, most of the time if I'm in singleplayer and press the key to open the editor, the mod will report "IBE Editor must be installed on the server in order to use it." Usually, I'd have to reinstall the mod and config for it to work again. Sometimes, the mod likes to be stubborn and not work after a reinstall. Sometimes it works right when I start up the game. IDK why it does this, but it does.

### TL;DR
IBE running in my environment likes to say a server on singleplayer doesn't have the mod when it does.

**Comment by Franckyi** [comment]
Thanks for reporting the issue. I'll see if I can reproduce the error on a similar setup.

**Comment by Franckyi** [comment]
@the3dbb Can you paste the  `debug.log` file after reproducing the issue?
